### PR TITLE
Add compilation support for nsProcess

### LINKED ISSUE #859: About hosting nsProcess.dll binary
https://github.com/corrados/jamulus/pull/841#issuecomment-761888194

Was merged now and should work correctly. However we currently get nsProcess.dll from a forum post which is clearly not the right way to do it. If we can host this file ourselves, this would be great.

### PR DESCRIPTION
**Pull request by ann0see**
Fixes #859

### CODE REVIEW

**Review by pljones — CHANGES_REQUESTED** [review]

File: windows/nsProcess/api.h
Comment (ann0see) [reply windows/nsProcess/api.h:11]: Good question...

File: windows/nsProcess/README_JamulusChanges.txt
Comment (ann0see) [reply windows/nsProcess/README_JamulusChanges.txt]: Fixed.

File: windows/nsProcess/api.h
Comment (ann0see) [reply windows/nsProcess/api.h:11]: Maybe here: https://github.com/kichik/nsis/blob/master/COPYING

Comment (pljones) [reply windows/nsProcess/api.h:11]: Probably needs a specific link or including.

Comment (ann0see) [reply windows/nsProcess/api.h:11]: I hope this is the correct file. But I think so.

Comment (ann0see) [reply windows/nsProcess/api.h:11]: I think this is the correct one: https://sourceforge.net/p/nsis/code/HEAD/tree/NSIS/trunk/COPYING

Comment (ann0see) [reply windows/nsProcess/api.h:11]: Ok. Added the file now.

**Comment by ann0see** [comment]
@nefarius2001 I assume, you'll need to include this PR in your PR too.

**Review by pljones — APPROVED** [review]

**Review by pljones — APPROVED** [review]

**Comment by pljones** [comment]
@softins any comments on this one?  Else @ann0see, do you want to approve and merge for 3.7.0?

**Comment by ann0see** [comment]
Probably we could merge this, but it's already included nefarius PR.

**Comment by ann0see** [comment]
Unfortunately, I can't approve it myself.

**Comment by softins** [comment]
I don't know enough about this module to comment knowledgeably. I assume the CI compiles it ok, and that it has been tested out on a Windows machine. Happy to approve if so.

**Comment by ann0see** [comment]
Yes. It compiles on Windows and also on GH Actions. I'm not responsible for all the deprecation warnings from this code. If we have time, we might fix them (but the code is not Jamulus related)

**Comment by pljones** [comment]
> 
> 
> Probably we could merge this, but it's already included nefarius PR.

Ah - @nefarius2001 can this be closed then?

**Comment by nefarius2001** [comment]
Sort of - I'd suggest closing when  #978 is merged

**Pull request merged**
